engine/src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -119,6 +124,44 @@ object JesBackend {
     case _ => false
   }
 
+
+  /**
+    * A short term (i.e. hotfix) attempt at allowing things in this file to not block on threads when needing
+    * to retry. Unabashed hodgepodge of TryUtil.retryBlock, the 'after' based retry in DataAccess and Rawls' Retry.
+    * Many hardcodes due to reproducing what the withRetry below was doing in conjuction with retryBlock
+    *
+    * Expect a more unifying solution in the non-hotfix branch
+    */
+  def retryAsync[A](f: => Future[A],
+                    logger: WorkflowLogger,
+                    failureMessage: String,
+                    backoff: SimpleExponentialBackoff = SimpleExponentialBackoff(5 seconds, 10 seconds, 1.1D),
+                    retries: Int = 10,
+                    isTransient: Throwable => Boolean = isTransientJesException,
+                    isFatal: Throwable => Boolean = isFatalJesException)
```

Comment:
Are `isTransient` and `isFatal` actually variable arguments? I couldn't find usage where they were anything but the defaults.